grails-app/migrations/changelog.groovy

```diff
@@ -63,4 +63,5 @@ databaseChangeLog = {
     include file: '2018-01-17-v440-addThreshold.groovy'
     include file: '2018-02-20-v450-dropMainUrl.groovy'
     include file: '2018-06-20-v460-removeScriptPageRelations.groovy'
+    include file: '2018-10-01-v500-job-label-optional.groovy'
```

Comment:
I'm afraid we get a little merge conflict here. Can you update your branch and solve the conflict?